docs/content/docs/dev/datastream/fault-tolerance/checkpointing.md

```diff
@@ -147,6 +154,11 @@ env.getCheckpointConfig.enableUnalignedCheckpoints()
 
 // sets the checkpoint storage where checkpoint snapshots will be written
 env.getCheckpointConfig.setCheckpointStorage("hdfs:///my/checkpoint/dir")
+
+// enable checkpointing with finished tasks
+val config = new Configuration()
+config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true)
+env.configure(config)
```

Comment:
What is the python way to enable this feature?

Reply:
https://issues.apache.org/jira/browse/FLINK-24199

Reply:
I encourage you to try add it yourself, sometimes it's not that difficult:
https://github.com/apache/flink/pull/17100/commits/e83081e3d734448f232e6f676d71c46a44c4081e

Here it might be a bit more tricky with using {{Configuration}}, but maybe there is already an example how to do it that you can just copy/paste/rename.

Reply:
I did add methods to the Python API in the past on multiple occasions. Here, I'd really prefer not to deal with it, as it deals with `Configuration`. 

1. First of all I am not able to run the python tests. Configuring the IDE to make them run, would take me quite some time.
2. In Java/Scala we use a type safe version of configuration `ReadableConfig`. I don't know how we should solve that in python.

Reply:
Ok, I get it.